neqo-transport/src/tparams.rs

```diff
@@ -128,7 +172,10 @@ pub enum TransportParameter {
 
 impl TransportParameter {
     fn encode(&self, enc: &mut Encoder, tp: TransportParameterId) {
-        qtrace!("TP encoded; type 0x{tp:02x} val {self:?}");
+        qtrace!(
+            "TP encoded; type {tp:?}(0x{:02x}) val {self:?}",
+            u64::from(tp.clone())
+        );
```

Comment:
Instead of `u64::from(tp.clone())`, how about implementing `Display` for `TransportParameterId`.

Also, given that `TransportParameterId` is a [field-less `enum`](https://doc.rust-lang.org/reference/items/enumerations.html#r-items.enum.fieldless) it can `derive` `Copy`, thus no need for the above call to `.clone()`.